dotCMS/src/main/java/com/dotcms/graphql/datafetcher/page/PagePropertiesDataFetcher.java

```diff
@@ -0,0 +1,21 @@
+package com.dotcms.graphql.datafetcher.page;
+
+import com.dotcms.util.CollectionsUtils;
+import com.dotmarketing.portlets.htmlpageasset.business.render.HTMLPageAssetRenderedAPIImpl.HTMLPageUrl;
+import com.dotmarketing.util.Logger;
+import graphql.schema.DataFetcher;
+import graphql.schema.DataFetchingEnvironment;
+
+public class PagePropertiesDataFetcher implements DataFetcher<Object> {
+    @Override
+    public Object get(final DataFetchingEnvironment environment) throws Exception {
+        try {
+            HTMLPageUrl pageUrl = environment.getSource();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'pageUrl' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5685344)